docs/maintain-guides-how-to-setup-sentry-node.md

```diff
@@ -0,0 +1,212 @@
+# keep the connection alive by sending a handshake every 21 seconds
+PersistentKeepalive = 21
+
+# Note: In this guide, we only set up 1 peer (public node)
+
+You need to do the previous steps (1 and 2) again in your **public node** but the `wg0.conf` configuration file will look like this:
+
+```bash
+[Interface]
+Address = 10.0.0.2/32
+PrivateKey = eCii0j3IWi4w0hScc8myUj5QjXjjt5rp1VVuqlEmM24=
+ListenPort = 51820
+SaveConfig = true
+
+# Validator
+[Peer]
+# replace this with the validator public key
+PublicKey = iZeq+jm4baF3pTWR1K1YEyLPhrfpIckGjY/DfwCoKns=
+# public ip address of the validator
+Endpoint = 55.321.234.4:51820
+# replace it with the validator interface address
+AllowedIPs = 10.0.0.1/32
+PersistentKeepalive = 21
+```
+### 4. Test-Connection
+
+If everything goes well, you are ready to test the connection.
+
+To start the tunnel interface, execute the following command in both your `validator` and `public node`.
+
+```bash
+wg-quick up wg0
+
+# The console would output something like this
+#[#] ip link add wg0 type wireguard
+#[#] wg setconf wg0 /dev/fd/63
+#[#] ip -4 address add 10.0.0.1/24 dev wg0
+#[#] ip link set mtu 1420 up dev wg0
+```
+
+You can check the status of the interface by running `wg` :
+
+```bash
+# Output
+ interface: wg0
+  public key: iZeq+jm4baF3pTWR1K1YEyLPhrfpIckGjY/DfwCoKns=
+  private key: (hidden)
+  listening port: 51820
+
+peer: Vdepw3JhRKDytCwjwA0nePLFiNsfB4KxGewl4YwAFRg=
+  endpoint: 112.223.334.445:51820
+  allowed ips: 10.0.0.2/32
+  latest handshake: 18 seconds ago
+  transfer: 580 B received, 460 B sent
+  persistent keepalive: every 25 seconds
+```
+
+You can then use `ping` to verify the connectivity between each other. 
+
+In case you want to update `wg0.conf`, run `wg-quick down wg0` to stop the interface first.
+
+### 5. Start your Sentry Node and Validator
+
+After you have started the `wg0` interface on your public node and validator, do spend a little bit of time to take a briefly look at the following description of those flags you are going to use. 
+
+`--sentry` - This would be required for your public node to be a authority as a observer that means it actually same as validator node but without holding keys / doing signing. And the difference between running a full node versus adding an extra `--sentry` flag is that running a authority node would have every data the validator is needed available, while full node might not have all the data the validator need to validate properly.
+
+`--reserved-nodes` - The node will try to connect to these nodes and always accept connections from them, but it will still connect and accept connections from other nodes as well. 
+
+`--reserved-only` - Only allows the connection from reserved nodes you defined
+
+Since we want to ensure that the sentry node would never reject a connection from our validator and this could happen if all the peer slots were full, `--reserved-nodes` will be required on the sentry to prevent that happen.
```

Comment:
```suggestion
Since we want to ensure that the sentry node would never reject a connection from our validator and this could happen if all the peer slots were full, `--reserved-nodes` will be required on the sentry to prevent that happening.
```